product_docs/docs/pgd/5/consistency/column-level-conflicts/02_enabling_disabling.mdx

```diff
@@ -0,0 +1,64 @@
+---
+navTitle: Enabling and disabling
+title: Enabling and disabling column-level conflict resolution
+deepToC: true
+redirects:
+  - /pgd/latest/bdr/column-level-conflicts/
+---
+
+!!! Note Permissions required
+Column-level conflict detection uses the `column_timestamps` type. This type requires any user needing to detect column-level conflicts to have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned.
+!!!
+
+The [bdr.alter_table_conflict_detection()](../../reference/conflict_functions/bdralter_table_conflict_detection) function manages column-level conflict resolution.
```

Comment:
bad link

Reply:
And although there was a function reference there which has moved, you really want some short text to explain the function - otherwise there's no context for the following example.

Reply:
I fixed the link, but I am not aware of a function reference that was moved.  The example is an example of enabling column-level conflict resolution on a table. We already say "The bdr.alter_table_conflict_detection() function manages column-level conflict resolution." What more were you thinking we should say for context?

Reply:
The reference link should be absolute. The text should say something like "by taking three parameters which control something something)

Reply:
Why should the reference link be absolute? I thought we used relative where possible.

Reply:
Except for the reference sections. And relative where possible *unless changing to other products*.

Reply:
Fixed this and went through and made sure all reference links were absolute.